### PR TITLE
Modify UserGuide.md

### PR DESCRIPTION
**Pull request by ZechariahTan**
* Add guide for Search command
* Format examples for better readability

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -66,16 +66,29 @@ Adds a person to the address book.
...
+>* `add n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01`
+>* `add n/Betsy Crowe t/friend e/betsycrowe@example.com a/Newgate Prison p/1234567 t/criminal`
 
 ### Listing all persons : `list`
 
 Shows a list of all persons in the address book.
 
 Format: `list`
 
+## Search for passengers: `search`
```
Comment (JoelHo) [inline docs/UserGuide.md]: This should be at heading level 3 to be consistent with the document.

File: docs/UserGuide.md [hunk 2]
```diff
@@ -89,7 +102,7 @@ Format: `edit INDEX [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [t/TAG]…​`
 * You can remove all the person’s tags by typing `t/` without
     specifying any tags after it.
 
-Examples:
+>**Examples**:
 *  `edit 1 p/91234567 e/johndoe@example.com` Edits the phone number and email address of the 1st person to be `91234567` and `johndoe@example.com` respectively.
 *  `edit 2 n/Betsy Crower t/` Edits the name of the 2nd person to be `Betsy Crower` and clears all existing tags.
```
Comment (JoelHo) [inline docs/UserGuide.md]: This should be in the quotes to match the other "examples" you changed.

File: docs/UserGuide.md [hunk 3]
```diff
@@ -66,16 +66,29 @@ Adds a person to the address book.
...
+Search passengers according to the constraints such as looking for female driver only or if they are available
+
+Format: `search CONSTRAINT`
+
+- Only one criteria can be search at a time
+- List of constraints that can be searched includes:
+  *availability*, *gender*
+
+> **Examples**:
+`search female` or `search available`
```
Comment (JoelHo) [inline docs/UserGuide.md]: Should this be in a different format to match the other "examples"? Additionally, perhaps the newline is not appearing as intended?

**Review by JoelHo — CHANGES_REQUESTED** [review]
LGTM, but minor cosmetic changes required to maintain consistency of the guide.

File: docs/UserGuide.md [hunk 4]
```diff
@@ -66,16 +66,28 @@ Adds a person to the address book.
...
+
+Search passengers according to the constraints such as looking for female driver only or if they are available
+
+Format: `search CONSTRAINT`
+
+- Only one criteria can be search at a time
+- List of constraints that can be searched includes:
+  *availability*, *gender*
+
+> **Examples**:`search female` or `search available`
```
Comment (markmcwong) [inline docs/UserGuide.md:89]: Just this `> **Examples**` has an extra space compared to every other ones in markdown, but visual wise does not change anything

**Review by markmcwong — COMMENTED** [review]
LGTM

**Pull request merged**